pkg/xds/generator/ingress_generator.go

```diff
@@ -160,7 +161,48 @@ func (_ IngressGenerator) destinations(
 		}
 	}
 
-	if len(policies[meshhttproute_api.MeshHTTPRouteType].GetItems()) > 0 {
+	addMeshHTTPRoutesDestinations(policies[meshhttproute_api.MeshHTTPRouteType],
```

Comment:
unfortunately it seems like gofumpt isn't strict enough here, can we avoid this kind of argument formatting and just put every argument on its own line? I think it's the style elsewhere in the code and I find it hard to read like this

Reply:
(Addressed) I don't have a strong opinion about this case, so I'm completely fine to follow approach suggested by you. It would be awesome if we could configure linter to check for it though 🤔.